packages/console/src/pages/SignInExperience/components/Preview.tsx

```diff
@@ -0,0 +1,82 @@
+import { Language } from '@logto/phrases';
+import { AppearanceMode, SignInExperience } from '@logto/schemas';
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import Card from '@/components/Card';
+import Select from '@/components/Select';
+import TabNav, { TabNavItem } from '@/components/TabNav';
+
+import * as styles from './Preview.module.scss';
+
+type Props = {
+  signInExperience: SignInExperience;
+};
+
+const Preview = ({ signInExperience }: Props) => {
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+  const [language, setLanguage] = useState<Language>(Language.English);
+  const [mode, setMode] = useState<AppearanceMode>(AppearanceMode.LightMode);
+  const [platform, setPlatform] = useState<string>('web');
+
+  // TODO: is a placeholder
+  const config = JSON.stringify({
+    ...signInExperience,
+    language,
+    mode,
+    platform,
+  });
+
+  return (
+    <Card className={styles.preview}>
+      <div className={styles.header}>
+        <div className={styles.title}>{t('sign_in_exp.preview.title')}</div>
+        <div className={styles.selects}>
+          <Select
+            value={language}
+            options={[
+              { value: Language.English, title: t('sign_in_exp.preview.languages.english') },
+              { value: Language.Chinese, title: t('sign_in_exp.preview.languages.chinese') },
+            ]}
+            onChange={(value) => {
+              setLanguage(value as Language);
+            }}
+          />
```

Comment:
nit: Select component should better be able to infer the value type

Reply:
created LOG-2415, since is not necessary in this PR.

Reply:
Done. #1071